src/regions/ca/steps.json

```diff
@@ -328,5 +328,40 @@
     "id": "outro1",
     "message": "outro1",
     "trigger": "showResults"
+  },
+  {
+    "id": "assessment-ON",
+    "message": "assessment-ON",
+    "trigger": "assessment-continue"
+  },
+  {
+    "id": "assessment-AB",
+    "message": "assessment-AB",
+    "trigger": "assessment-continue"
+  },
+  {
+    "id": "assessment-continue",
+    "message": "assessment-continue",
+    "trigger": "assessmentOptions"
+  },
+  {
+    "id": "assessmentOptions",
+    "options": [
+      {
+        "label": "yes",
+        "value": true,
+        "trigger": "askAgeRange"
+      },
+      {
+        "label": "no",
+        "value": false,
+        "trigger": "outro2"
+      }
+    ]
+  },
+  {
+    "id": "outro2",
+    "message": "outro2",
+    "trigger": ""
```

Comment:
Btw, you shouldn't have empty triggers, but rather mark those steps with `end: true` -- just make sure the rules / class system can handle it :)

Reply:
@vicrep do you have an example? trying to find one in the docs but cannot

Reply:
Look at user step here: https://lucasbassetti.com.br/react-simple-chatbot/#/docs/steps it's not specified in docs but same can be done for a message step

Reply:
Thanks!

Reply:
Thanks!